File: hw/chalan/u3/hw27GUIsOperation/MODULOS/Python/StoreModules/domain/domain.py
DOMAIN = {
    "title": "Laptop Store",
    "collection": "laptops",

    "idField": "serial",

    "fields": [
        {"name": "serial", "type": str},
        {"name": "brand", "type": str},
        {"name": "model", "type": str},
        {"name": "cpu", "type": str},
        {"name": "ram_gb", "type": int},
        {"name": "storage_gb", "type": int},
        {"name": "screen_inches", "type": float},
        {"name": "price", "type": float},
        {"name": "stock", "type": int}
    ],

    "priceField": "price",
    "stockField": "stock"
}

File: hw/chalan/u3/hw27GUIsOperation/MODULOS/JS/FruitsModules/src/config/domain.js
export default {
  title: 'Fruit Store',
  collection: 'fruits',
  fields: [
    {
      name: 'name',
      type: 'string',
      required: true,
      pattern: /^[A-Za-z\s]+$/
    },
    {
      name: 'price',
      type: 'number',
      required: true
    },
    {
      name: 'stock',
      type: 'int',
      required: true
    }
  ]
}
